src/trees/MultiResolutionAnalysis.cpp

```diff
@@ -117,6 +193,14 @@ template <int D> void MultiResolutionAnalysis<D>::setupFilter() {
     }
 }
 
+/** @returns double (double-precision floating point number) 
```

Comment:
you should write down what the function returns, not its type. The type will be provided by doxygen.